packages/opam-compiler/opam-compiler.0.1.0/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+synopsis: "Plugin to create switches using custom compilers"
+description:
+  "This plugin can manage switches using various sources for compilers, such as git branches, github PRs, etc"
+maintainer: ["Etienne Millon <me@emillon.org>"]
+authors: ["Etienne Millon <me@emillon.org>"]
+license: "ISC"
+tags: ["flags:plugin"]
```

Comment:
this should be `flags: plugin` to be explicit I think.

Reply:
(that is, a direct field, not just the tags entry)

Reply:
hmm I stole that from opam-publish:

https://github.com/ocaml/opam-repository/blob/54b0223dcacb6311aa12f5342c34e1be684a79a3/packages/opam-publish/opam-publish.2.0.1/opam#L9

but publish uses just flags:

https://github.com/ocaml/opam-repository/blob/54b0223dcacb6311aa12f5342c34e1be684a79a3/packages/publish/publish.0.3.4/opam#L23

I think this was correctly picked, but I'll double check the opam docs and sources

Reply:
indeed, opam just looks at `flags:`. I missed this because it's only considered when opam knows about this plugin, but it's not installed. So I didn't test this case. Anyway, that should be ok now.